css/templates/lune/components/_chapter.css

```diff
@@ -16,10 +16,17 @@
   letter-spacing: 0;
   /* Position the chapter vertically in the centre */
   position: absolute;
-  top: 50%;
+  top: calc((100% - var(--navbar-height)) / 2);
```

Comment:
Fixes the vertical position of the chapter name. It now takes into account the navbar height